### PR TITLE
blogs: add new posts for 2021

### PR DESCRIPTION
**Pull request by cpanato**
## Description
blogs: add new posts for 2021

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
No was a request to update the page with posts about openfaas

- [ ] My issue has received approval from the maintainers or lead with the `design/approved` label

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ It would be great to hear from you especially if you have any of the above and w
 
 | Blog/repo name and description                                          | Author       | Site     | Date        |
 |-------------------------------------------------------------------------|--------------|----------|-------------|
+| [Kubernetes in Docker (KinD) で OpenFaaS を動かす！](https://yoh1496.hatenablog.com/entry/2021/05/17/182207) | ishiguro-yo | yoh1496.hatenablog.com | 17-May-2021 |
```
Comment (alexellis) [inline community.md:94]: We are missing the ones from the OpenFaaS blog like Kim and Istio

**Review by alexellis — CHANGES_REQUESTED** [review]
Few more to add

File: community.md (hunk 1 above)
Comment (cpanato) [reply community.md:94]: done, thanks for the review and feedback

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**